src/views/tables/PositionsTable.tsx

```diff
@@ -201,11 +201,7 @@ const getPositionsTableColumnDef = ({
               sign={getNumberSign(size?.current)}
               fractionDigits={stepSizeDecimals}
             />
-            <Output
-              type={OutputType.Fiat}
-              value={notionalTotal?.current}
-              fractionDigits={tickSizeDecimals}
-            />
+            <Output type={OutputType.Fiat} value={notionalTotal?.current} fractionDigits={2} />
```

Comment:
nit: we can use the constant `USD_DECIMALS` here! or it might already be the case for `Output` by default when type = OutputType.Fiat

Reply:
oo ya ur so right its just the default!